client/web/src/enterprise/batches/create/CreateOrEditBatchChangePage.tsx

```diff
@@ -271,6 +273,9 @@ const EditPage: React.FunctionComponent<EditPageProps> = ({ batchChange, refetch
     )
 
     const [filters, setFilters] = useState<WorkspacePreviewFilters>()
+    const [modal, setModal] = useState<boolean>(false)
```

Comment:
I try to give boolean variable names a name prefixed with `is` to make it clearer what sort of value it is:

```suggestion
    const [isDownloadSpecModalOpen, setIsDownloadSpecModalOpen] = useState<boolean>(false)
```